client/reader/stream/style.scss

```
@@ -645,7 +646,8 @@
 			}
 
 			@include breakpoint-deprecated( "<660px" ) {
-				margin: 0 16px;
+				margin-left: 16px;
+				margin-right: 16px;
```

Seems like we can use margin-inline here.

```suggestion
				margin-inline: 16px 16px;
```